docs/community/Issue-Triaging.md

```diff
@@ -0,0 +1,108 @@
+|[`polish`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | a feature could be improved, but not necessarily a bug|
+|[`*question`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | we should direct questions to StackOverflow|
+
+### Feature Areas Labels
+
+Each issue must have a **feature area** label. 
+
+|Feature Area|Description|
+|---|---|
+|[`rendering`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | Rendering |
+|[`shader`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | Shader |
+|[`styling`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | Styling |
+
+### Platform Labels
+
+Each issue must have a **platform** label. 
+
+|Platform|Description|
+|---|---|
+|[`ios-simulator`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | iOS Simulator|
+|[`ios-device`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | Physical iOS device|
+|[`android-simulator`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | Android Simulator |
+|[`android-device`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | Physical Android device|
+|[`macos-x86`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | Mac OS |
+|[`macos-arm`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | M-series chips Mac OS |
+|[`linux`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | Linux |
+|[`qt`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | Qt |
+|[`node`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | Node.JS |
+|[`all-platforms`](https://github.com/maplibre/maplibre-gl-native/labels/xx) | Across all platforms |
+
+### Important Issues
+
+The TSC reviewers may add `important` labels to issues that
+- Critical security vulnerabilities
+- Stability or performance issue that makes a feature unusable
+
+### Other labels
+
+If issues are suitable for beginners we may add the `good-first-issue` label and we add code pointers that help beginners to get started with a PR.
+
+Sometime, we get issues that we can't or don't have the time to reproduce due to the complexity or time requirements of the setup but that we indeed suspect to be issues. We label those issues with `investigation-wanted`. What we are looking for is help in reproducing and analyzing the issue.
+
+
+## Community Review
+
+### Accepting issue
+Community members can "up-vote" an issue (adding a GitHub +1/"👍" reaction to the issue description). If an issue surpasses the 20 up-votes in 60 days, the issue will be accepted.
+
+### Closing issue
+
+Community members can also close an issue after reviewing it, for the following reasons:
+
+|Reason|Label|
+|---|---|
+|It's a duplicate of another issue. | `*duplicate`|
+|What is described is the designed behavior. | `*as-designed`|
+|The issue is a developer question, should be direct to Github Discussion or our Slack channel| `*dev-question`|
```

Comment:
auto-converting all dev/user questions to discussions seem excessive tbh.  Plus issues are fairly easy to search, manually or via the searching engine, so might make sense to answer it first, tag it as a question, and only then close it.  Plus a question is usually an indication of a missing/bad documentation :)